eventarc/audit-storage/src/main/java/com/example/cloudrun/EventController.java

```diff
@@ -37,7 +36,7 @@
 
   @RequestMapping(value = "/", method = RequestMethod.POST)
   public ResponseEntity<String> receiveMessage(
-      @RequestBody PubSubBody body, @RequestHeader Map<String, String> headers) {
+      @RequestBody Object body, @RequestHeader Map<String, String> headers) {
```

Comment:
Even though this isn't the point of the sample, removing PubSubBody feels like it makes this sample not useful for continued learning and building.

Reply:
This sample literally does not receive Pub/Sub requests. It receives Cloud Audit Log requests. In this case, the audit log is for a Cloud Storage event.

Thus, it's incorrect to have Pub/Sub in this sample. The sample does not use this body, so the tests have always passed. If we wanted a different POJO, we cannot include a [`LogEntryData`](https://github.com/googleapis/google-cloudevents-java/blob/master/google-cloudevent-types/src/main/java/com/google/events/cloud/audit/v1/LogEntryData.java) POJO until that library is finished and published.

We have a Pub/Sub sample for Pub/Sub events.

Reply:
You could build your own POJO for the time being like Pub/Sub.  From this code, I have no idea what it accepts so if possible at least add a content type. We shouldn't use "Object", `Map<String, String>` would at least be useable.

Reply:
I've changed the type to a Map per suggestion.

---

The JSON payload is large for an Audit Log. The POJO would have 14 classes (see link above), unless we construct a custom POJO for this type.

The actual contents of an Audit Log is very strange, hard to use, and is not useful for this sample of an audit log for a Cloud Storage event.

I don't think there's value in adding/maintaining a POJO in Java for this specific event type during public preview and would rather have a generic type until we publish https://github.com/googleapis/google-cloudevents-java/tree/master/google-cloudevent-types/src/main/java/com/google/events/cloud/audit/v1